source/utils.ts

```diff
@@ -38,3 +38,13 @@ export function set(object: UnknownRecord, path: string, value: unknown) {
 
   object[last] = value;
 }
+
+export function getAddCachedTranslationsJS(
```

Comment:
How about moving all of the code-generation helpers to one file (`getJS`, etc.)? `utils.ts` is bundled in the client's code as well but it's not needed there.